paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptor.kt

```diff
@@ -177,8 +177,12 @@ internal class DefaultIntentConfirmationInterceptor @Inject constructor(
         shippingValues: ConfirmPaymentIntentParams.Shipping?,
         setupForFutureUsage: ConfirmPaymentIntentParams.SetupFutureUsage?,
     ): NextStep {
+        var attribution = paymentMethodCreateParams.attribution + "deferred-intent"
+        if (intentConfiguration.paymentMethodTypes.isNullOrEmpty()) {
```

Comment:
`paymentMethodTypes` can’t be null, so we should replace `isNullOrEmpty` with `isEmpty`.